### ISSUE TITLE
[Feature] Check for _clone_excluded_m2o_or_o2m_fields is None instead of truthy

### ISSUE THREAD

**Issue by saschahofmann** [tj-django/django-clone#549]
### Is this feature missing in the latest version?

- [X] I'm using the latest release

### Is your feature request related to a problem? Please describe.

We want to enforce the use of the __clone_excluded_*_ fields and set them on our `BaseModel`:

```python
class BaseModel(CloneMixin, models.Model):

    _clone_excluded_fields = []
    _clone_excluded_m2m_fields = []
    _clone_excluded_m2o_or_o2m_fields = []
    _clone_excluded_o2o_fields = []
```

The problem now is that empty lists are of course falsy. Line 491 in _mixins/clone.py_ then return false and no field is copied.

```python
if any(
    [
        f.get_accessor_name() in self._clone_m2o_or_o2m_fields,
>      self._clone_excluded_m2o_or_o2m_fields
        and f.get_accessor_name()
        not in self._clone_excluded_m2o_or_o2m_fields,
    ]
)
```

At least for me that wasn't the expected behaviour.

### Describe the solution you'd like?

This should be easily handled by checking for `self._clone_excluded_* is not None`.

### Describe alternatives you've considered?

I'll just make them `self._clone_excluded_* = ['']` for now.

### Anything else?

Happy to make a PR if you're ok with my proposal

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jackton1** [comment]
@saschahofmann Sure I’ll welcome a PR but I’m still somewhat puzzled about your proposed solution, are you suggesting we use a default of `[‘’]` instead of an empty list

**Comment by saschahofmann** [comment]
Nope! That's my workaround for now. I suggest we make None the default and check for `is None` instead of the truthyness of the argument.

**Comment by saschahofmann** [comment]
It'd be non backwards compatible if someone would overwrite the default itself with an empty list but I would think that's rare and acceptable?
